fastapi/responses.py

```diff
@@ -21,3 +22,10 @@ class ORJSONResponse(JSONResponse):
     def render(self, content: Any) -> bytes:
         assert orjson is not None, "orjson must be installed to use ORJSONResponse"
         return orjson.dumps(content)
+
+
+class ModelResponse(JSONResponse):
```

Comment:
Also add a stringdoc to the class , cause it could be confusing with a "response_model" and this class exist in the framework

"""
This class let you respond with a pydantic model without the data validation ...
"""

Reply:
Done!